packages/components/psammead-play-button/src/index.jsx

```diff
@@ -61,11 +61,14 @@ const PlayButton = ({
   service,
   title,
   onClick,
+  guidanceMessage,
 }) => {
-  const hiddenText =
+  const hiddenText = `${guidanceMessage || ''} Play ${type}, ${
     datetime && duration && durationSpoken
-      ? `Play ${type}, "${title}", ${durationSpoken}`
-      : `Play ${type}, "${title}"`;
+      ? `"${title}", ${durationSpoken}`
+      : `"${title}"`
+  } `;
```

Comment:
The snapshots are revealing that there is a lot of additional whitespace being generated by the addition of this property. We should be trimming that before it gets rendered - this can be done with `.trim()`. E.G.:

```
const hiddenText = `${guidanceMessage || ''} Play ${type}, ${
  datetime && duration && durationSpoken
    ? `"${title}", ${durationSpoken}`
    : `"${title}"`
} `.trim();
```